tools/scripts/ffi.toml

```diff
@@ -146,6 +147,18 @@ rm ../../../../../target/debug/libicu_capi_staticlib_test.a
 exec --fail-on-error make
 '''
 
+[tasks.test-js]
```

Comment:
what about executing the wasm build?

Reply:
the node module's build script builds the wasm file, same as the c++ example make files build `icu_capi.a`

Reply:
but I'm still trying to figure out what exactly is happening in wasm.toml